creation/web_base/cvmfs_umount.sh

```diff
@@ -40,18 +40,19 @@ use_cvmfsexec=$(gconfig_get GLIDEIN_USE_CVMFSEXEC "$glidein_config")
 # TODO: int or string? if string, make the attribute value case insensitive
 #use_cvmfsexec=${use_cvmfsexec,,}
 
-if [[ $use_cvmfsexec -ne 1 ]]; then
-    "$error_gen" -ok "$(basename $0)" "umnt_msg1" "Not using cvmfsexec; skipping cleanup."
-    exit 0
-fi
-
 # get the glidein work directory location from glidein_config file
 work_dir=$(gconfig_get GLIDEIN_WORK_DIR "$glidein_config")
 # $PWD=/tmp/glide_xxx and every path is referenced with respect to $PWD
 
 # source the helper script
-# TODO: Is this file somewhere in the source tree? use: # shellcheck source=./cvmfs_helper_funcs.sh
-. $work_dir/cvmfs_helper_funcs.sh
+# shellcheck source=./cvmfs_helper_funcs.sh
+. "$work_dir"/cvmfs_helper_funcs.sh
+
+if [[ $use_cvmfsexec -ne 1 ]]; then
+    "$error_gen" -ok "$(basename $0)" "umnt_msg1" "On-demand CVMFS not requested or not used; skipping cleanup."
+    loginfo "On-demand CVMFS provisioning not requested or not used; skipping cleanup."
```

Comment:
Switch the order, usually error_gen is the last instruction before exiting

Reply:
Updated as suggested.